lib/Widgets/TransmissionListEntry.vala

```diff
@@ -41,10 +41,11 @@ public class Drop.Widgets.TransmissionListEntry : Gtk.Grid {
      * Creates a new transmission entry and it's ui.
      */
     public TransmissionListEntry () {
-        build_ui ();
+        this.build ();
```

Comment:
is it really needed to rename the method to build() ? They are all private, I doup that's neccessary.